operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -300,6 +279,18 @@ private static boolean isCurrentPodValid(V1Pod build, V1Pod current) {
       return false;
     }
 
+    if (areUnequal(
+        volumesWithout(current.getSpec().getVolumes(), ignoring), build.getSpec().getVolumes()))
+      return false;
+
+    if (areUnequal(current.getSpec().getImagePullSecrets(), build.getSpec().getImagePullSecrets()))
+      return false;
+
+    if (areUnequal(getCustomerLabels(current), getCustomerLabels(build))) return false;
```

Comment:
Except resource version and restart versions, labels set by operator will not change, right?

Reply:
Correct.